packages/reqresp/src/response/index.ts

```diff
@@ -77,24 +77,28 @@ export async function handleRequest<Req, Resp>({
           }
         });
 
-        logger.debug("Req  received", {...logCtx, body: protocol.renderRequestBody?.(requestBody)});
+        logger.debug("Req  received", logCtx);
 
-        const requestCount = protocol?.inboundRateLimits?.getRequestCount?.(requestBody) ?? 1;
+        // TODO: Consider counting max count by request for byRange and byRoot
+        // const requestCount = protocol?.inboundRateLimits?.getRequestCount?.(requestBody) ?? 1;
+        const requestCount = 1;
```

Comment:
this is a side effect of this change regarding rate limiter. Can we decode requests before this as before? Otherwise we should track this in a github issue

Reply:
Re-added